### PR TITLE
WRN-1385: Migrate VideoPlayer unit tests to Testing Library

### PR DESCRIPTION
**Pull request by stanca-pop-lgp**
Enact-DCO-1.0-Signed-off-by: Stanca Pop stanca.pop@lgepartner.com

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Migrated unit tests from Enzyme to Testing Library for VideoPlayer component.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRN-1385

### CODE REVIEW

**Review by alexandrumorariu — APPROVED** [review]
LGTM.

File: VideoPlayer/tests/Video-specs.js [hunk 1]
```diff
@@ -27,237 +20,250 @@ describe('VideoPlayer.Video', () => {
...
 
-			subject.setProps({});
+			rerender(
+				<Video />
+			);
 
-			expect(getSourceNode(subject)).toBe(preload);
-			expect(getPreloadNode(subject)).toBe(source);
+			expect(newSource).toBe(preload);
+			expect(newPreload).toBe(source);
```
Comment (daniel-stoian-lgp) [inline VideoPlayer/tests/Video-specs.js:181]: Here you are checking values saved before the new rerender. You will need to read the nodes again and compare them tp newSource and newPreload

Comment (stanca-pop-lgp) [reply VideoPlayer/tests/Video-specs.js:181]: Thank you!

File: VideoPlayer/tests/Video-specs.js [hunk 2]
```diff
@@ -27,237 +20,255 @@ describe('VideoPlayer.Video', () => {
...
-			expect(getPreloadNode(subject)).toBe(source);
-		});
+			const newSource = screen.getByTestId('video-id');
+			const newPreload = screen.getByTestId('video-id').nextElementSibling;
 
+			expect(newSource).toBe(preload);
+			expect(newPreload).toBe(source);
+		});
 
 		test('should not swap nodes on re-render after swapping `source` and `preloadSource`', () => {
```
Comment (stanca-pop-lgp) [inline VideoPlayer/tests/Video-specs.js]: This test has been deleted. It cannot be migrated to react testing library. Re-rendering <Video /> with no props doesn't allow accessing the initial value for those props like in the enzyme test.

**Pull request merged**